api/package.json

```diff
@@ -77,12 +77,12 @@
         "eslint-plugin-import": "^2.25.4",
         "eslint-plugin-prettier": "^4.0.0",
         "husky": "^8.0.3",
-        "jest": "^27.4.7",
+        "jest": "^29.5.0",
         "lint-staged": "^13.1.2",
         "msw": "^0.36.4",
         "nodemon": "^2.0.15",
         "prettier": "^2.5.1",
-        "puppeteer": "10.1.0",
+        "puppeteer": "^20.3.0",
```

Comment:
this also needs to be reverted to 10.1.0 until we investigate upgrading puppeteer in a separate ticket because this is a spike really